O21.Game/Localization/Help.fs

```diff
@@ -0,0 +1,30 @@
+module O21.Localization.Help
+
+open O21.Game.Documents
+open System.Collections.Generic
+open System.Threading.Tasks
+open System.IO
+open System.Threading
+
+type HelpRequest =
+    | EnglishHelp of CancellationToken
+    | RussianHelp of (unit -> Task<DocumentFragment array>)
+
+let private readMarkdown(file: string, cancellationToken: CancellationToken) =
+    task {
+        use sr = new StreamReader (file)
+        let fragments = List<DocumentFragment> 0
+        while not sr.EndOfStream do
+            let! line = sr.ReadLineAsync cancellationToken
+            fragments.Add (match line with
+                            | "---" -> (DocumentFragment.Text(Style.Normal, "____________________________________________"))
+                            | str when (str.Length > 0 && str[0] = '#') -> DocumentFragment.Text(Style.Bold, str.Substring(1))
+                            | str -> (DocumentFragment.Text(Style.Normal, str)))
+            fragments.Add (DocumentFragment.NewParagraph)
+        return fragments.ToArray()
```

Comment:
The original help also included several images. Care to add them back?

I'd suggest just adding a couple of special marks like `![imageA][]` and `![imageB][]` and performing a simple replacement. But you could choose to do more if you wish :)

Reply:
![image](https://github.com/ForNeVeR/O21/assets/26199107/8ddf3abb-dd8a-4f12-af43-a16aa7156a0a)
Added but have to mention that occasionally I see "Failed to load texture" in logs and the following image:
![image](https://github.com/ForNeVeR/O21/assets/26199107/5083780d-160d-43f0-b167-0dc0bd30ca8d)

Reply:
Seems to work well for me. Check if you aren't affected by #92.

Reply:
![CyrillicPathFontBug](https://github.com/ForNeVeR/O21/assets/26199107/1ea7624b-4af2-4e45-8ac7-0bea3d677ddd)
I'm affected but I've done no changes to the font usage. I just wrote "Style.Normal" or "Style.Bold". So if somebody fixes #92 my code will work fine without any changes

Reply:
> occasionally I see "Failed to load texture" in logs and the following image

I've got it: textures are unable to load from a background thread in Raylib. I'll update everything to work synchronously for now; we'll work on async loading in another scope.